test/Turn-test.js

```diff
@@ -0,0 +1,80 @@
+const expect = require('chai').expect;
+
+const Turn = require('../src/Turn');
+const Card = require('../src/Card');
+
+describe('Turn', function() {
+
+  it('should be a function', function() {
+    const turn = new Turn();
+    expect(Turn).to.be.a('function');
+  });
+
+  it('should be an instance of Turn', function() {
+    const turn = new Turn();
+    expect(turn).to.be.an.instanceof(Turn);
+  });
+
+  it('should store a guess', function() {
+    const turn = new Turn('array');
+    expect(turn.guess).to.equal('array');
+  });
+
+  it('should store a card', function() {
+    const card = new Card(1, 'What allows you to define a set of related information using key-value pairs?', ['object', 'array', 'function'], 'object');
+
+    const turn = new Turn('array', card);
+    expect(turn.card).to.equal(card);
+  });
```

Comment:
I need to make sure I know the correct whitespace for tests, and standardize mine. This seems wonky.